Gemfile

```diff
@@ -23,7 +24,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  if beaker_version
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```

Comment:
Let's keep it broken is not really a good statement guys. It's too easy to add `:systemtests` around this - it will make it consistent with other modules as well.